utils/create_command_docs.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+
+import importlib
+from pathlib import Path
+from typing import Any
+from typing import Generator
+from typing import List
+from typing import NamedTuple
+from typing import Optional
+
+import click
+
+TEMPLATE = """
```

Comment:
Is it worth leveraging Jinja2?

Reply:
Yes. Resolved in https://github.com/uktrade/copilot-tools/pull/83/commits/074403a9c2867a89ef12e585a955ad11013e839d.